src/storage/src/source/persist_source.rs

```diff
@@ -70,16 +67,17 @@ where
             return;
         }
 
-        let persist_location = PersistLocation {
-            consensus_uri: consensus_uri,
-            blob_uri: blob_uri,
-        };
-
-        let persist_client = persist_location.open().await?;
-
-        let mut read = persist_client
-            .open_reader::<Row, Row, Timestamp, Diff>(shard_id)
-            .await?;
+        let (_, mut read) = crate::persist_cache::open::<
+                (),
+                SourceData,
```

Comment:
was the old code wrong by making the key a `Row`? we are past enveloping, so we always had no key value here right?

Reply:
that's right